action.yaml

```diff
@@ -7,7 +7,7 @@ branding:
 inputs:
   root-folder:
     required: false
-    default: ""
+    default: "./"
```

Comment:
Can we leave the default empty and then only do the `cd` if it is set to something non-empty?